lib/rules/media-query-no-invalid/index.mjs

```diff
@@ -143,6 +158,31 @@ const rule = (primary) => {
 	};
 };
 
+/**
+ * @param {import('@csstools/media-query-list-parser').MediaQuery|import('@csstools/media-query-list-parser').GeneralEnclosed} node
+ * @param {Object|undefined} secondaryOptions
+ * @returns {boolean}
+ */
+function shouldIgnoreNode(node, secondaryOptions) {
+	let ignored = false;
+
+	if (secondaryOptions && 'ignoreFunctions' in secondaryOptions) {
```

Comment:
question
nit

Why not `if (secondaryOptions?.ignoreFunctions) {`?

Reply:
I've fixed it.
[chore: add strict type definition for SecondaryOptions](https://github.com/stylelint/stylelint/pull/8060/commits/ae03274b5b8221bf7d49fab86a95bbb5e8e6e545)